redirect.json

```
@@ -26,14 +53,129 @@
     }
   },
   {
-    "redirect": "config_flow_start",
-    "description": "allow starting a config flow",
-    "introduced": "2021.3",
+    "redirect": "automations",
+    "description": "show your automations",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "scenes",
+    "description": "show your scenes",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "scripts",
+    "description": "show your scripts",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "helpers",
+    "description": "show your helper entities",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "tags",
+    "description": "show your tags",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "lovelace_dashboards",
+    "description": "show your Lovelace dashboards",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "lovelace_resources",
+    "description": "show your Lovelace resources",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "people",
+    "description": "show your people",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "zones",
+    "description": "show your zones",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "users",
+    "description": "show your users",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "general",
+    "description": "show your general Home Assistant settings",
+    "introduced": "2021.3"
+  },
+  {
+    "redirect": "server_controls",
+    "description": "allow to control your server",
```

Lets avoid the word "allow", almost looks like the link reboot your machine or somethin

```suggestion
    "description": "Show your server controls",
```